fsd_utils/locale_selector/get_lang.py

```diff
@@ -20,6 +20,11 @@ def get_lang():
     # get locale from cookie if set
     locale_from_cookie = request.cookies.get(CommonConfig.FSD_LANG_COOKIE_NAME)
     if locale_from_cookie:
+        if locale_from_cookie not in ["cy", "en"]:
+            current_app.logger.warning(
+                f"Invalid language code {locale_from_cookie}. Supported codes are 'cy' and 'en'."
+            )
+            return "en"
```

Comment:
Should this remove or update the cookie if it's a bad value? Otherwise every future request from the user will still emit this warning.

Reply:
Can we not log this as a warning please? I don't think we really care that they've got a non-supported language do we? We'll potentially get a lot of noise from this

Reply:
removed warning messages, as there is no response object we can't delete or update cookies..